pyproject.toml

```diff
@@ -39,7 +39,7 @@ min-public-methods = 1
 [tool.coverage.report]
 fail_under = 100
 show_missing = true
-omit = ["kedro/templates/*", "kedro/framework/cli/hooks/specs.py", "kedro/framework/hooks/specs.py", "kedro/extras/datasets/tensorflow/*", "kedro/extras/datasets/holoviews/*", "tests/*"]
+omit = ["kedro/templates/*", "kedro/extras/logging/color_logger.py", "kedro/framework/cli/hooks/specs.py", "kedro/framework/hooks/specs.py", "kedro/extras/datasets/tensorflow/*", "kedro/extras/datasets/holoviews/*", "tests/*"]
```

Comment:
I've added the color logger here. I actually have no idea why we were getting 100% coverage before because this file has never been fully tested... But given that we're going to deprecate it I decided just to add it to the coverage ignore rather than write tests for it.

Reply:
Isn't there a `test_color_logger.py`? It's probably not related but seen this from Kiyo yesterday. https://921kiyo.com/code-coverage-level-with-pytest-cov/

Reply:
There is, but it only tests how the color logger handles `INFO` level messages. Ideally it would go through every level of logging and test them all. I don't know how it was passing coverage before given none of the other levels are tested 🤔

Reply:
Interesting post on the different coverage levels, I didn't know about that before! I wonder what we'd get if we added `--cov-branch` to our tests.